components/card/index.en-US.md

```diff
@@ -32,6 +32,7 @@ A card can be used to display content related to a single subject. The content c
 | hoverable | Lift up when hovering card | boolean | false |
 | loading | Shows a loading indicator while the contents of the card are being fetched | boolean | false |
 | tabList | List of TabPane's head. | Array&lt;{key: string, tab: ReactNode}> | - |
+| size | Size of card | default\|middle\|small | `default` |
```

Comment:
```
| size | Size of card | `default` \| `middle` \| `small` | `default` |
```

Reply:
Hello, thanks for your review! I tried this first, but it does not look very good:

<img width="834" alt="screen shot 2018-11-18 at 3 43 53 am" src="https://user-images.githubusercontent.com/139536/48665918-f85e2880-eae9-11e8-9d5b-a86d4ddea58e.png">

But I will revert this change